h3/src/server.rs

```diff
@@ -400,7 +403,7 @@ where
     fn poll_control(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Error>> {
         while let Poll::Ready(frame) = self.inner.poll_control(cx)? {
             match frame {
-                Frame::Settings(_) => trace!("Got settings"),
+                Frame::Settings(w) => trace!("Got settings > {:?}", w),
```

Comment:
here I want to intercept when the browser sent the enable_webtransport setting

Reply:
I am working on that now